src/azure/msal/msalAzureController.ts

```diff
@@ -201,6 +202,20 @@ export class MsalAzureController extends AzureController {
                 } catch (ex) {
                     this._vscodeWrapper.showErrorMessage(ex);
                 }
+            }
+            if (getErrorMessage(ex).includes(AzureConstants.multiple_matching_tokens_error)) {
+                const response = await this._vscodeWrapper.showErrorMessage(
+                    LocalizedConstants.ConnectionDialog.multipleMatchingTokensError(
```

Comment:
If `account?.displayInfo?` ends up being undefined, we should have another message that doesn't add the account info so it looks cleaner.